src/blog/2023/06/community-news-06.md

```diff
@@ -0,0 +1,64 @@
+
+ Read about the details of FlowForge 1.8 in our release announcement. 
+
+## Upcoming events
+
+### Building Node-RED Applications for Scalability and High Availability
+
+Our June webinar will focus on the new FlowForge 1.8 of running high availability Node-RED applications. Marian Demme, FlowForge Product Manager, will lead this session and share practical insights and best practices to show how FlowForge can unlock the true potential of Node-RED in large-scale deployments. 
+
+[Sign-up today](https://flowforge.com/webinars/2023/building-scalable-ha-node-red/) to join us on June 22. 
+
+### Build an Edge-to-Cloud Solution with the MING Stack
+
+On June 27, FlowForge is doing a webinar with our friends at InfluxDB. A great opportunity to see how easy it is to use Node-RED and InfluxDB to send data from the edge to the cloud.  [Sign-up today](https://www.influxdata.com/resources/build-an-edge-to-cloud-solution-with-the-ming-stack/?utm_source=partner&utm_medium=referral&utm_campaign=2023-06-27_Webinar_FlowForge-NodeRED&utm_term=speaker).
+
+## From our Blog
+
+- [Bringing High Availability to Node-RED](https://flowforge.com/blog/2023/05/bringing-high-availability-to-node-red/) - FlowForge CTO discusses the strategy for delivering high availability in the FlowForge platform.
+
+- Two articles featuring how to connect Modbus data with Node-RED:
+    - [Using Node-RED to Visualize Industrial Production Data via Modbus](https://flowforge.com/blog/2023/05/visualize-production-data-via-modbus-in-node-red/)
+    - [Best Practices Integrating a Modbus Device With Node-RED](https://flowforge.com/blog/2023/05/integrating%20modbus%20with%20node-red/)
+
+- [Node-RED Tips - Dashboard Edition](https://flowforge.com/blog/2023/06/3-quick-node-red-tips-7/) - A new set of Node-RED quick tips that are focused on using Node-RED Dashboard.
+
+- [Persisting chart data in Node-RED Dashboards](https://flowforge.com/blog/2023/05/persisting-chart-data-in-node-red/)
+
+- [Node-RED Community Survey Results](https://flowforge.com/blog/2023/05/node-red-community-survey-results/) - A quick summary of the Node-RED Community Survey results.
+
+- [FlowForge 1.7 Now Available with Remote Node-RED Editor Access](https://flowforge.com/blog/2023/05/flowforge-1-7-released/)
+
+
+
+## From the Community
+
+Gerrit Riessen has published a list of [Pros and Cons for using Node-RED](https://gorenje.medium.com/fourteen-for-fourteen-against-why-i-love-hate-and-connect-with-node-23797f9466ec). It is a pretty comprehensive list so check it out. FlowForge is working to address some of the cons in Gerrit's list, specifically software delivery pipelines and the ability to deploy out to many end points.
+
+
+
+## Join Our Team
+FlowForge is expanding our team. Check out the current openings:
+
+* [DevOps Engineer](https://boards.greenhouse.io/flowforge/jobs/4796271004)
+
```

Comment:
```suggestion
* [Sales Representative](https://boards.greenhouse.io/flowforge/jobs/4843566004)
```

Reply:
Should we not have both jobs?

Reply:
added Sales Rep job.  It wasn't actually listed when I checked. Now I see it.